go.mod

```diff
@@ -84,6 +85,7 @@ require (
 )
 
 replace (
+	github.com/argoproj/gitops-engine => github.com/jessesuen/gitops-engine v0.1.3-0.20201030112610-d0d513da07ef
```

Comment:
This will be reverted after https://github.com/argoproj/gitops-engine/pull/177 is merged